bench/Gimlet.adoc

```diff
@@ -0,0 +1,87 @@
+= Running OPTE Benchmarks on a Gimlet (pair)
+
+The kernel module benchmarks can be run on a pair of lab gimlets connected over their `cxgbe` interfaces.
+Today, SN9/SN14 are set up in this configuration.
+
+== Image construction
+When https://github.com/oxidecomputer/helios[building a helios image], we need to include some extra packages and pull in the latest (host-installed) version of OPTE.
+The below patch adds these packages behind a conditional flag:
+
+[source, diff]
+----
+diff --git a/image/templates/gimlet/ramdisk-01-os.json b/image/templates/gimlet/ramdisk-01-os.json
+index a063784..c81974f 100644
+--- a/image/templates/gimlet/ramdisk-01-os.json
++++ b/image/templates/gimlet/ramdisk-01-os.json
+@@ -105,6 +105,14 @@
+             "/driver/developer/amd/zen"
+         ] },
+
++        { "t": "pkg_install", "with": "netperf", "pkgs": [
++            "/network/test/iperf",
++            "/driver/network/opte",
```

Comment:
If we do go with adding a `netperf` feature, seems like you could omit `opte` here and just pass `-F optever=xx`.

Reply:
Sounds reasonable. I should also remove `sparse`, since we're using omicron1 here.